lms/templates/commerce/receipt.underscore

```diff
@@ -0,0 +1,97 @@
+<div class="wrapper-content-main payment-confirmation-step">
+  <article class="content-main">
+    <h3 class="title">
+      <%= gettext( "Thank you! We have received your payment." ) %>
+    </h3>
+
+    <% if ( receipt ) { %>
+    <ul class="list-info">
+      <li class="info-item payment-info">
```

Comment:
There is only 1 item, this doesn't look like a list to me. Lists with only 1 item are an accessibility issue.